package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "neomad-styleguide",
+  "version": "1.0.0",
+  "description": "Neomad styleguide, built with atomic design mindset",
+  "scripts": {
+    "dev": "postcss ./content/styles/styleguide.css -o ./content/dist/styleguide.css --watch",
+    "prod": "postcss ./content/styles/main.css -o ./content/dist/main.css"
```

Comment:
Why calling it _styleguide.css_ in dev and _main.css_ in prod?

Reply:
Styleguide is for styleguide + main and prod is just main. If you think it’s weird or not logic, we could change. You prefer something like dev.css and prod.css ??

Reply:
my idea was to have a _styleguide_ dedicated file only in the styleguide repo and _main_ or whatever the name used in both repos

Reply:
ok works for me!

Reply:
@newick ok for _styleguide.css_ but about _main.css_ (whatever the name ^^) just one called do by neomad.org could be better, so just one file could be avoid us different version and keep all the CSS here